apps/condo/domains/organization/integrations/sbbol/sync/requestTransactions.js

```diff
@@ -141,8 +141,7 @@ async function requestTransactionsForDate ({ userId, bankAccounts, context, stat
                     organization: { id: organizationId },
                     account: { id: bankAccount.id },
                     importId: transactionProperty.importId,
-                    number: transactionProperty.number,
-                    date: transactionProperty.date,
```

Comment:
Could you please rename `transactionProperty` to avoid confusion with `Property` schema.
Maybe, `transactionAttrs`?